canonical-web-and-design/ubuntu.com
# Update yarn.lock

## Done

- Update the yarn.lock file

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the site runs without compaining about the new middleware added for /advantage
